templates/taxi/driver_list.html

```diff
@@ -8,6 +9,11 @@ <h1>
       </a>
     </h1>
 
+  <form action="" method="get" class="form-inline" style="margin-bottom: 2px" >
+    {{ search_form|crispy }}
+    <input style="margin-left: 2px" class="btn btn-secondary" type="submit" value="Search">
+  </form>
```

Comment:
Please create a separate template for the search form and include it where needed.